npx hedera-local start throws an error

Even when docker is started and all docker containers are removed - start throws the errors below:

- <img width="436" alt="image (1)" src="https://user-images.githubusercontent.com/101559469/165477664-492fb33e-5e26-461c-a979-f45961a5dd5c.png">
- <img width="422" alt="image" src="https://user-images.githubusercontent.com/101559469/165477676-46ba960c-97ac-4950-a746-b57e5f314daa.png">

hi, can you help me with this ?

npm install --save-dev 'hedera-local'
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/'hedera-local' - Not found
npm ERR! 404
npm ERR! 404 ''hedera-local'@' is not in this registry.
npm ERR! 404 This package name is not valid, because
npm ERR! 404 1. name can no longer contain special characters ("~'!()")
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\Vipul\AppData\Local\npm-cache_logs\2022-04-24T14_14_39_786Z-debug-0.log

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@hashgraph/hedera-local",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Developer tooling for running Local Hedera Network (Consensus + Mirror Nodes).",
   "homepage": "https://github.com/hashgraph/hedera-local-node",
   "repository": "github.com:hashgraph/hedera-local-node",

